environment.yml

```diff
@@ -1,14 +1,17 @@
 name: adkf-jax
 channels:
   - conda-forge
+  - nvidia
 dependencies:
   - python=3.8
   - pip
   # Standard data science stuff
   - numpy
   - tqdm
   # ML libraries
+  - jaxlib=*=*cuda*
```

Comment:
Why did you put `*cuda*`? What happens if you omit it? What about a CPU-only install?